uint32  action 

uint32 PUSH_VERTEX = 1 
uint32 PULL_VERTEX = 2
uint32 ASSIGN_DESCRIPTOR_VERTEX = 3
uint32 PUSH_EDGE = 4
uint32 PULL_EDGE = 5
uint32 ASSIGN_DESCRIPTOR_EDGE = 6
uint32 GET_VERTEX_LIST = 7
uint32 GET_EDGE_LIST = 8
uint32 GET_DESCRIPTOR_LINKS = 9
uint32 GET_NEIGHBOR_VERTICES = 10
uint32 GET_OUTGOING_EDGES = 11
